src/RoomsSDKAdapter.js

```diff
@@ -159,6 +161,25 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
     return this.getRoomObservables[ID];
   }
 
+  /**
+   * Returns an observable that emits room data of the recently created room
+   * Observable will complete after one emission.
+   *
+   * @param {Room} room  Information about the room to create.
+   * @returns {external:Observable.<Room>} Observable stream that emits room data.
+   * @memberof RoomsAdapter
+   */
+  createRoom(room) {
+    logger.debug('ROOM', room, 'createRoom()', ['called with', {room}]);
```

Comment:
the second param of the logger functions is normally the ID of the resource. Here, we don't have yet an ID, so you can put `undefined` instead. The content of the `room` parameter is already logged in the 'called with' section.